themes/catppuccin-mauve.json

```diff
@@ -1,1583 +1,1587 @@
-                    "type.interface": {
-                        "color": "#f9e2af",
-                        "font_style": null,
-                        "font_weight": null
-                    },
-                    "type.super": {
-                        "color": "#f9e2af",
-                        "font_style": null,
-                        "font_weight": null
-                    },
-                    "variable": {
-                        "color": "#cdd6f4",
-                        "font_style": null,
-                        "font_weight": null
-                    },
-                    "variable.member": {
-                        "color": "#cdd6f4",
-                        "font_style": null,
-                        "font_weight": null
-                    },
-                    "variable.parameter": {
-                        "color": "#fab387",
-                        "font_style": "italic",
-                        "font_weight": null
-                    },
-                    "variable.special": {
-                        "color": "#dcb8d5",
-                        "font_style": "italic",
-                        "font_weight": null
-                    },
-                    "variant": {
-                        "color": "#f38ba8",
-                        "font_style": null,
-                        "font_weight": null
-                    }
-                }
-            }
+  "$schema": "https://zed.dev/schema/themes/v0.1.0.json",
+  "name": "Catppuccin",
+  "author": "Andrew Tec <andrewtec@enjoi.dev>",
+  "themes": [
+    {
+      "name": "Catppuccin Latte",
+      "appearance": "light",
+      "style": {
+        "border": "#ccd0da",
+        "border.variant": "#9658eb",
+        "border.focused": "#7287fd",
+        "border.selected": "#8839ef",
+        "border.transparent": "#40a02b",
+        "border.disabled": "#acb0be",
+        "elevated_surface.background": "#e6e9ef",
+        "surface.background": "#eff1f5",
+        "background": "#eff1f5",
+        "element.background": "#dce0e8",
+        "element.hover": "#9ca0b04d",
+        "element.active": "#8839ef33",
+        "element.selected": "#ccd0da99",
+        "element.disabled": "#9ca0b0",
+        "drop_target.background": "#8839ef66",
+        "ghost_element.background": null,
+        "ghost_element.hover": "#acb0be4d",
+        "ghost_element.active": "#acb0be80",
+        "ghost_element.selected": "#9ca0b059",
+        "ghost_element.disabled": "#9ca0b0",
+        "text": "#4c4f69",
+        "text.muted": "#5c5f77",
+        "text.placeholder": "#acb0be",
+        "text.disabled": "#9ca0b0",
+        "text.accent": "#8839ef",
+        "icon": "#4c4f69",
+        "icon.muted": "#8c8fa1",
+        "icon.disabled": "#9ca0b0",
+        "icon.placeholder": "#acb0be",
+        "icon.accent": "#8839ef",
+        "status_bar.background": "#dce0e8",
+        "title_bar.background": "#dce0e8",
+        "title_bar.inactive_background": "#dce0e8",
```

Comment:
keeping the color as catppuccin's "crust" color makes sense. to make the "inactive" state obvious, maybe we can keep some of the "transparent" coloring a bit, but not so transparent since that seems to be the fallback behavior 

<img width="331" alt="image" src="https://github.com/user-attachments/assets/008a9166-d7fa-4930-8007-744df0d0e860">